mmyolo/models/dense_heads/yolov6_head.py

```diff
@@ -342,12 +342,13 @@ def loss_by_feat(
         else:
             target_labels, target_bboxes, target_scores, fg_mask = \
                 self.assigner(
```

Comment:
The most reasonable ordering should be `pred_` in front, so it is better to unify all the codes